src/Settings/Builders/Hsc2Hs.hs

```diff
@@ -38,18 +43,16 @@ getCFlags = do
     mconcat [ remove ["-O"] (cArgs <> getStagedSettingList ConfCcArgs)
             , getStagedSettingList ConfCppArgs
             , cIncludeArgs
-            , getPkgDataList CppArgs
-            , getPkgDataList DepCcArgs
+            , getConfiguredCabalData ConfCabal.ccOpts
+            -- XXX: is cppOpts correct here?
```

Comment:
Instead of asking a question, perhaps, it would be better to write "We don't know if `cppOpts` here is correct".

Reply:
Fixed the comment.